src/styles/sidebar/components/datalist.scss

```diff
@@ -0,0 +1,77 @@
+@use "../../variables" as var;
+
+.datalist {
+  position: relative;
+}
+
+.datalist__items {
+  &:before {
+    /* creates a small outlined triangle above the drop down menu */
+    background-color: inherit;
+    border: inherit;
+    border-top-left-radius: 0.125rem;
+    clip-path: polygon(0 0, 100% 0, 0% 100%, 0% 100%);
```

Comment:
I'll take with Jared about this, but we went through some trouble to arrive at this css fix. I'll bring it up with him today.

Reply:
Thanks. I don't completely object to using CSS for this, but we should know why we're using this vs SVG.

Since this element is not critical to the usability of the component, I think it would be an acceptable solution for IE / Edge if it just didn't appear in those browsers. Maybe [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) could be used to test?